itou/cities/models.py

```diff
@@ -54,7 +54,7 @@ class Meta:
             # https://docs.djangoproject.com/en/dev/ref/contrib/postgres/search/#trigram-similarity
             # https://docs.djangoproject.com/en/dev/ref/contrib/postgres/indexes/#ginindex
             # https://www.postgresql.org/docs/11/pgtrgm.html#id-1.11.7.40.7
-            GinIndex(fields=["name"], name="cities_city_name_gin_trgm", opclasses=["gin_trgm_ops"])
+            GinIndex(fields=["normalized_name"], name="cities_city_norm_name_gin_trgm", opclasses=["gin_trgm_ops"])
```

Comment:
Soyons urbains :grin:, surtout qu'on le fait à 2 endroits.
```suggestion
            GinIndex(fields=["normalized_name"], name="%(class)s_normalized_name_gin_trgm", opclasses=["gin_trgm_ops"])
```

Reply:
Le soucis c'est que le nom des index est limité à 30 caractères :-/

Reply:
Yup, j'ai tronqué les suffix pour que ça rentre :)
Le plus triste c'est que cette limite de 30 vient d'[Oracle](https://github.com/django/django/blob/5.1.6/django/db/models/indexes.py#L15-L17) :unamused: 
Et que Postgresql gère par défaut jusqu'à 63 caractères: https://www.postgresql.org/docs/15/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS
Je serais presque tenté d'aller adapter `Index.max_name_length` :eyes:

Reply:
au bucher oracle